### PR TITLE
workflow: Create publish pipeline

### PR DESCRIPTION
**Pull request by kevindavee**

### CODE REVIEW

File: .github/workflows/bintray.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.11
+    - name: Grant execute permission for gradlew
+      run: chmod +x gradlew
+    - name: Upload SDK
+      env:
+        BINTRAY_USER: ${{ secrets.BINTRAY_USER }}
+        BINTRAY_API_KEY: ${{ secrets.BINTRAY_API_KEY }}
+      run: BINTRAY_USER=$BINTRAY_USER BINTRAY_API_KEY=$BINTRAY_API_KEY ./gradlew bintrayUpload
```
Comment (stanleynguyen) [inline .github/workflows/bintray.yml:25]: I dont think you would need to `BINTRAY_USER=$BINTRAY_USER BINTRAY_API_KEY=$BINTRAY_API_KEY`?

Comment (kevindavee) [reply .github/workflows/bintray.yml:25]: Oh, ya. You're right. The env itself has been injected into that session right. I'll remove it

**Review by stanleynguyen — APPROVED** [review]

**Comment by stanleynguyen** [comment]
Please don't forget to set the secrets

**Comment by kevindavee** [comment]
It's done before I create this deployment script :)

**Pull request merged**